### PR TITLE
feat: adds constants dir

### PR DESCRIPTION
**Pull request by noahamac**
refactors constants structure

### CODE REVIEW

File: src/utils/constants/diagram.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+ */
+
+// Diagram zoom
```
Comment (bryans99) [inline src/utils/constants/diagram.ts:21]: How come this don't get docs?

File: src/utils/constants/diagram.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+ /**
+  * the vertical padding between tables of the same degree
+  */
+export const TABLE_VERTICAL_PADDING = 5
+ /**
+  * the vertical step each degree will take from the base table
+  */
+export const TABLE_DEGREE_STEP = -3
+
+export const DIAGRAM_IGNORED_MODELS = ['system__activity']
```
Comment (bryans99) [inline src/utils/constants/diagram.ts:56]: Doc?

**Review by bryans99 — APPROVED** [review]
LGTM

A little bit of inconsistency about use of docs but nothing to stop the merge

**Comment by noahamac** [comment]
It felt silly docstringing synonyms of the constant's name... Will fix in another pass

**Pull request merged**